src/file_path.rb

```diff
@@ -148,6 +148,14 @@ def log
         '/var/log/metalware'
       end
 
+      def asset_template(type)
+        File.join('/var/lib/metalware/repo/assets', "#{type}.yaml")
+      end
+
+      def asset_final(name)
```

Comment:
Also can you change the name of this method to just `asset`. Whilst this is the final location for the saving process, they will be loaded again latter.

Calling `FilePath.asset` is descriptive enough for our purposes.